integration-test/common/conftest.py

```diff
@@ -0,0 +1,76 @@
+import pytest
```

Comment:
A file level comment explaining the purpose of this library and how it works would be helpful.

Reply:
Right. Added a file-level comment at the beginning of this.